api/commons/parameters/nonRequiredAccountId.yaml

```diff
@@ -0,0 +1,9 @@
+in: query
+name: accountId
+example: "act-123456789"
+description: >
+  The ID of the account associated with your token.
+  Can be ommitted if there is a single account in the organization.
```

Comment:
Is this a Spot convention to make the accountId param non-required if there's a single account?

I think this default behavior is dangerous for API users, as as an account gets added to the organization (which could happen at a random time, by someone who doesn't manage/use Ocean for Spark), then all of a sudden the Ocean Spark API calls would start failing because this parameter becomes required.

There are two alternatives:
* Make it always required (my preference).
* Make it default to the "oldest account".

Reply:
I believe it's actually the "oldest account". I wrote "single account" because I didn't want to say a mistake 😆 
@alextarasov-spot can you enlighten us on how this currently works?

Reply:
I found somewhere in the doc that it is indeed the oldest account that is used.
@jystephan this is not a behavior we control, it is implemented at the optimizer level. Our upstream services require an account ID.
Changed the doc accordingly!